cmd/podman/varlink/io.podman.varlink

```diff
@@ -144,10 +144,11 @@ type PsOpts (
     last: ?int,
     latest: ?bool,
     noTrunc: ?bool,
-	pod: ?bool,
-	quiet: ?bool,
-	sort: ?string,
-	sync: ?bool
+    pod: ?bool,
+    quiet: ?bool,
+    sort: ?string,
+    sync: ?bool,
+    size: ?bool
```

Comment:
Can we order these alphabetically? Size before sort and sync?

Reply:
fixed